### ISSUE TITLE
Plugin brakes after packer compile

### ISSUE THREAD

**Issue by dainiusm07** [ZhiyuanLck/smart-pairs#16]
The issue is that when I'm doing packer compile plugin brakes neovim and It becomes unusable because it doesn't let to write any of 'pairs' symbols. After restarting neovim everything works as usual.

Errors that I'm getting:

```
packer.nvim: Error running config for smart-pairs: ...nvim/site/pack/packer/opt/smart-pairs/lua/pairs/init
.lua:198: expect length of pair table to greater than 1, bug the length is 0
```

```
Error detected while processing BufNew Autocommands for "*":                                              
E5108: Error executing lua ...nvim/site/pack/packer/opt/smart-pairs/lua/pairs/init.lua:207: bad argument #
1 to 'pairs' (table expected, got nil)                                                                    
stack traceback:                                                                                          
        [C]: in function 'pairs'                                                                          
        ...nvim/site/pack/packer/opt/smart-pairs/lua/pairs/init.lua:207: in function 'set_keymap'         
        [string ":lua"]:1: in main chunk
```

```
E5108: Error executing lua ...nvim/site/pack/packer/opt/smart-pairs/lua/pairs/init.lua:372: the left brack
et ( is not defined                                                                                       
stack traceback:                                                                                          
        [C]: in function 'error'                                                                          
        ...nvim/site/pack/packer/opt/smart-pairs/lua/pairs/init.lua:372: in function 'get_opts'           
        ...nvim/site/pack/packer/opt/smart-pairs/lua/pairs/init.lua:377: in function 'get_ignore'         
        ...nvim/site/pack/packer/opt/smart-pairs/lua/pairs/init.lua:438: in function 'clean_all'          
        ...nvim/site/pack/packer/opt/smart-pairs/lua/pairs/init.lua:497: in function 'has_right_start'    
        ...vim/site/pack/packer/opt/smart-pairs/lua/pairs/enter.lua:27: in function 'type_aux'            
        ...vim/site/pack/packer/opt/smart-pairs/lua/pairs/enter.lua:61: in function 'type'                
        [string "..."]: in function 'on_keymap'                                                           
        ...hare/nvim/site/pack/packer/opt/nvim-cmp/lua/cmp/core.lua:144: in function 'callback'           
        ...m/site/pack/packer/opt/nvim-cmp/lua/cmp/utils/keymap.lua:112: in function <...m/site/pack/packe
r/opt/nvim-cmp/lua/cmp/utils/keymap.lua:108>                                                              
        ...m/site/pack/packer/opt/nvim-cmp/lua/cmp/utils/keymap.lua:242: in function <...m/site/pack/packe
r/opt/nvim-cmp/lua/cmp/utils/keymap.lua:241> 
```

```
E5108: Error executing lua ...nvim/site/pack/packer/opt/smart-pairs/lua/pairs/init.lua:361: the left brack
et ( is not defined                                                                                       
stack traceback:                                                                                          
        [C]: in function 'error'                                                                          
        ...nvim/site/pack/packer/opt/smart-pairs/lua/pairs/init.lua:361: in function 'get_right'          
        ...m/site/pack/packer/opt/smart-pairs/lua/pairs/bracket.lua:162: in function 'type_left'          
        [string ":lua"]:1: in main chunk
```

Btw, thanks this awesome plugin!

**Comment by ZhiyuanLck** [comment]
Duplicate of #3, this will not be fixed for now.

**Comment by smjonas** [comment]
Just fyi, this was fixed in #19
